3d_segmentation/unetr_segmentation_3d.ipynb

```diff
@@ -0,0 +1,4317 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #68.    model.load_state_dict(torch.load(os.path.join(root_dir, "best_metric_model.pth")))
</pre><p>Could you please help delete the output of this cell? It's too line in the notebook and you already plot the loss curve and validation metrics curve, I think we don't need to keep the detailed log.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Fixed.